src/mars.d

```diff
@@ -241,6 +240,31 @@ extern (C++) void genCmain(Scope* sc)
     rootHasMain = sc._module;
 }
 
+/**
+ * Recursevely call semantic3() on import tree nodes.
+ */
+private void semantic3OnDependencies(Module m, bool[string] importArray)
+{
+    import std.conv : to;
```

Comment:
calling phobos inside dmd is a nono.
Since you are already using AA's you might as well use the dmd internal hash-table for strings.
it's in root.